conf/common/test/deepep_low_latency.toml

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
+# Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name = "deepep_low_latency"
+description = "DeepEP MoE Benchmark - Low Latency Mode"
+test_template_name = "DeepEPBenchmark"
+
+[cmd_args]
```

Comment:
1. Let's cleanup comments.
2. We can't to put these configs under test/common as they are not publicly available. See how DDLB was added in #711, for example.